src/internal/scheduled/schedulePromise.ts

```diff
@@ -1,21 +1,23 @@
+/** @prettier */
 import { Observable } from '../Observable';
 import { SchedulerLike } from '../types';
-import { Subscription } from '../Subscription';
 
 export function schedulePromise<T>(input: PromiseLike<T>, scheduler: SchedulerLike) {
-  return new Observable<T>(subscriber => {
-    const sub = new Subscription();
-    sub.add(scheduler.schedule(() => input.then(
-      value => {
-        sub.add(scheduler.schedule(() => {
-          subscriber.next(value);
-          sub.add(scheduler.schedule(() => subscriber.complete()));
-        }));
-      },
-      err => {
-        sub.add(scheduler.schedule(() => subscriber.error(err)));
-      }
-    )));
-    return sub;
+  return new Observable<T>((subscriber) => {
+    return scheduler.schedule(() =>
```

Comment:
nitpick: The scheduled action's subscription has already been added to the subscriber, so returning it isn't necessary.

Reply:
In this case, I don't think that's true. This bit was just removing an unnecessary `Subscription`.

Reply:
🤦‍♂️